config.go

```diff
@@ -443,6 +443,47 @@ func (c *Config) GetAll(alias, key string) ([]string, error) {
 	return all, nil
 }
 
+// HostsAliases returns all hosts aliases in the configuration that match the
+// given pattern, or nil if none are present.
```

Comment:
I'd prefer to return `[]string{}` if there's no match - we shouldn't treat 0 matches differently than 1 match.

Reply:
ok, changed.